config/routes.rb

```diff
@@ -70,7 +70,7 @@
   # for the docs-fargate prefix
   scope ":prefix", constraints: { prefix: /docs|docs-fargate/}, defaults: { prefix: "docs" } do
     get "tutorials",    to: redirect { |params| "/#{params[:prefix]}/tutorials/getting-started" }, status: 302
-    get "integrations", to: redirect { |params| "/#{params[:prefix]}/integrations/github-enterprise" }, status: 302
```

Comment:
Maybe this merge should result in two lines?

```
get "integrations", to: redirect { |params| "/#{params[:prefix]}/integrations/github" }, status: 302
get "integrations", to: redirect { |params| "/#{params[:prefix]}/integrations/github-enterprise" }, status: 302
```

Reply:
wait, that advice doesn't make sense :thinking:

Reply:
:man_facepalming: entirely unrelated to the  fargate stuff. Sorry for the noise. Fixed in b3b3d9d8c6